gh_pages/_downloads/ros-industrial-training-setup.sh

```diff
@@ -21,7 +21,7 @@ sudo apt update -y
 # ROS1 install
 sudo apt install -y ros-melodic-desktop
 sudo apt install -y python-wstool python-catkin-tools
-sudo apt install -y ros-melodic-perception ros-melodic-urdf-tutorials ros-melodic-moveit
```

Comment:
The package `ros-melodic-urdf-tutorials` does not exist. The correct package name is `ros-melodic-urdf-tutorial`, but it is already installed as a dependancy on another package.